src/internals/SimpleChildComponent.js

```diff
@@ -36,12 +35,11 @@ class SimpleChildComponent extends EventComponent {
     } else {
       const googleMapsClassName = this.constructor._GoogleMapsClassName;
       if (!objectPath.has(googleMapsApi, googleMapsClassName)) {
-        warning(false,
```

Comment:
This warning method is not longer included globally and it needs to be added including another package, so instead of including that package I used `console.warn()` for now, at least until we update or change the `react-google-maps` package